app/assets/stylesheets/application.css.scss

```diff
@@ -26,38 +29,29 @@ body {
   background: $medium;
   color: $dirty-white;
   font-family: 'Lato', Arial, sans-serif;
-  font-weight: 300;
+  font-weight: 400;
 }
 
 .main {
   padding-top: 20px;
 }
 
-legend {
-
-  a {
-    font-weight: lighter;
-  };
+.reminder {
+  background: linear-gradient($medium, #9dbcbc);
```

Comment:
Color literals like `#9dbcbc` should only be used in variable declarations; they should be referred to via variable everywhere else.